src/v2_1/messages/use_priority_charging.rs

```diff
@@ -0,0 +1,40 @@
+use super::{CustomData, PriorityChargingStatusEnum, StatusInfo};
+
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct UsePriorityChargingRequest {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub custom_data: Option<CustomData>,
+    pub transaction_id: String,
```

Comment:
This one should have

```rust
#[validate(length(max = 36), custom(function = "validate_identifier_string"))]
```

Reply:
Might be other places as well, if it's an `identifiedString` in the specification